<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color: #f7f2ea;
color: #7a0606;
font-family: VCR,Courier New,Courier,Lucida Sans Typewriter,Lucida Typewriter,monospace;
}

QPushButton{
	font-size: 14px;
	opacity: 0.6;
	border-radius: 2px; 
	background-color: #d0d0d0;
	width: 40%;
	color: #7a0606;
	height: 40%;
}

QPushButton:hover{
	opacity:1;
}</string>
  </property>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>441</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>271</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;p style=&quot;text-align: center; font-size: 22px&quot;&gt;&lt;b&gt;YEEZY&lt;br&gt;ONLINE&lt;br&gt;STORE&lt;/b&gt;&lt;/p&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font-size:14px</string>
             </property>
             <property name="text">
              <string>USERNAME:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_user">
             <property name="styleSheet">
              <string notr="true">font-size:14px;</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">font-size: 14px;</string>
             </property>
             <property name="text">
              <string>PASSWORD:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_pass">
             <property name="styleSheet">
              <string notr="true">font-size:14px</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">color:solid black;</string>
         </property>
         <property name="text">
          <string>LOGIN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
